### PR TITLE
Link thing

### LINKED ISSUE #120: Getting compiling error when using molecule in macos
`e: Module "moe.tlaster:precompose-molecule" has a reference to symbol app.cash.molecule/launchMolecule|-7305106088441339121[0]. Neither the module itself nor its dependencies contain such declaration.`

### LINKED ISSUE #124: Migrate tests to `commonTest`
We should be testing on all platforms

### PR DESCRIPTION
**Pull request by JakeWharton**
Closes #120. Closes #124.

Not sure what fixed the problem, but it fixed itself.

### CODE REVIEW

**Review by mattprecious — APPROVED** [review]

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -21,6 +21,11 @@ jobs:
           distribution: 'zulu'
           java-version: 11
 
+      # For watchOS x86 simulator. Remove with Kotlin 1.8.
+      - uses: maxim-lobanov/setup-xcode@v1
```
Comment (hfhbd) [inline .github/workflows/build.yaml:25]: Do you have a youtrack ticket? I got the same error with Xcode 14.

Comment (JakeWharton) [reply .github/workflows/build.yaml:25]: It's already fixed on their `master` so I didn't bother. The file is called `ExecutorService.kt` in their repo.

Comment (hfhbd) [reply .github/workflows/build.yaml:25]: Okay, but thanks for the workaround! Hopefully, 1.8.0 beta will be out soon.

Comment (JakeWharton) [reply .github/workflows/build.yaml:25]: Unfortunately it looks like it's currently only in Kotlin 1.8.20 according to GitHub, but perhaps it was cherry-picked 🤞

**Pull request merged**